Sources/BraintreePayPalMessaging/BTPayPalMessagingClient.swift

```diff
@@ -0,0 +1,151 @@
+import UIKit
+import SwiftUI
+import PayPalMessages
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Use `BTPayPalMessagingClient` to display PayPal messages to promote offers such as Pay Later and PayPal Credit to customers.
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public class BTPayPalMessagingClient: UIView {
+
+    // MARK: - Properties
+
+    public weak var delegate: BTPayPalMessagingDelegate?
+
+    var apiClient: BTAPIClient
+
+    // MARK: - Initializers
+
+    ///  Initializes a PayPal Messaging client.
+    /// - Parameter apiClient: The Braintree API client
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+
+        super.init(frame: .zero)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Method
+
+    /// Creates a view to be displayed to promote offers such as Pay Later and PayPal Credit to customers.
+    /// - Parameter request: an optional `BTPayPalMessagingRequest`
+    /// - Note: use `BTPayPalMessagingDelegate` protocol to receive notifications for events
+    public func createView(_ request: BTPayPalMessagingRequest = BTPayPalMessagingRequest()) {
```

Comment:
Another nit/idea - this name could be something that signals it is an asynchronous operation? Would `loadView` be more clear? Or `requestView()`?

Reply:
🤔 `requestView(request)` reads a bit weird to me. We could also consider moving the request into the `init` and making this method internal. I know we threw around that idea earlier but could be one way to clear that up. Could also use `start` which is more similar to how we launch UI based flows on PPCP, so `BTPayPalMessageView.start(request)`?

Reply:
irl-ed and decided to go with `start`: https://github.com/braintree/braintree_ios/pull/1153/commits/322008fb14f0b7ed6ca2f342b3332edd85568afa